activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Fix bug where `ActiveSupport::Timezone.all` would fail when tzinfo data for
+    any timezone defined in `ActiveSupport::MAPPING` is missing.
```

Comment:
How would this happen in real world? All zones defined in that constant we know exists.

Reply:
They do exist if the `tzdata` system package is current, but when it is not `TZInfo` does not know anything about them and raises an exception itself. Before 078421bacba178eac6a8e607b16f3f4511c5d72f `AS::Timezone.all` just omitted those timezones.